apps/settings/js/panels/languages/languages.js

```diff
@@ -37,8 +40,24 @@ define(function(require) {
           f.localeFormat(d, _('shortTimeFormat'));
       }
     },
-    onInit: function(panel) {
+    showMoreLanguages: function() {
+      SettingsCache.getSettings(function(result) {
+        var version = result['deviceinfo.os'];
+        return new MozActivity({
```

Comment:
I added the `return` statement to work around JSLint's warning `W031`: _Do not use 'new' for side effects._.  Do you prefer me to use `/* jshint -W031 */` here?